### PR TITLE
clear apc cache because reasons

### PR DESCRIPTION
**Pull request by laftho**

### CODE REVIEW

**Comment by funvill** [comment]
> because reasons

:P

**Comment by laftho** [comment]
Yea the APC cache for NGINX / PHP7 IPM is pretty aggressive.. mostly this isn't important because the loading of service methods in nomos is ephemeral and they are more sane with cache reloads but some base components, such as app.php / include.php or any of the modules, etc tend to get cached indefinitely and make it really frustrating to figure out why your shit don't work on the live server!

Also we've been having a problem where the server.log file pointer wasn't being updated during deploy, that's because the cached app.php and associated file logger handle was still pointing to the old deploy.

Restarting PHP7 IPM will clear this cache and force these modules to be properly reloaded upon a new deploy - very much a good thing.

tl;dr; because reasons ;)
